clinics/clinics_test.go

```diff
@@ -1,12 +1,13 @@
 package clinics_test
 
 import (
+	clinicsTest "github.com/tidepool-org/clinic/clinics/test"
```

Comment:
What are you using for a code formatter? Shouldn't this be down below with the other github.com/tidepool entries?

Not a blocker of course.

Reply:
I'm using IntelliJ which uses go fmt.